### PR TITLE
Add project info

### PR DESCRIPTION
**Pull request by danielpassos**

### CODE REVIEW

**Comment by matzew** [comment]
I think the "deprecation" hint should be good enough, in combination w/ the links to the new locations,

**Comment by lfryc** [comment]
@matzew the idea behind putting project info here was direct people to docs, mailing lists and other important places without need of one more step.

But I agree the complete project info here possibly does not make sense here (e.g. build and license)

**Comment by matzew** [comment]
yeah, a click on the library/repo in question, gives the relevant info.

**Comment by danielpassos** [comment]
@matzew @lfryc Removed the project-info commit and added a deprecation section. wdyt?

File: README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # aerogear-android
 
+## Deprecation
+
 This project was split in some modules. Take a look at the new modules. If you need access our old versions. See our [release](https://github.com/aerogear/aerogear-android/releases) tags
```
Comment (secondsun) [inline README.md]: s/some/separate

Comment (secondsun) [inline README.md]: old versions**, s**ee

Comment (secondsun) [inline README.md]: Should probably be Deprecated not Deprecation

Comment (danielpassos) [inline README.md]: Done!

**Comment by lfryc** [comment]
:+1: go ahead!

**Pull request merged**